### PR TITLE
Dropdown for About

### PR DESCRIPTION
**Pull request by Anoninnyc**
### What github.com/wevote/WebApp/issues does this fix?
#769

### Changes included this pull request?
Dropdown for about button and 4 Routes

### CODE REVIEW

**Comment by Anoninnyc** [comment]
@DaleMcGrew Pls pull down my "dropdown" branch and let me know if you'd like me to make any changes.

File: src/js/Root.jsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import FAQ from "./routes/More/FAQ";
 import Friends from "./routes/Friends";
 import GetStarted from "./routes/Intro/GetStarted";
 import GuidePositionList from "./routes/Guide/GuidePositionList";
+import HowToUse from "./routes/More/howtouse";
 import Intro from "./routes/Intro/Intro";
```
Comment (DaleMcGrew) [inline src/js/Root.jsx]: Hi @Anoninnyc, the Component file name should be "HowToUse.jsx", but the route that appears in the browser (elsewhere in the code) should be all lower case.

**Comment by DaleMcGrew** [comment]
Hi Chris, the files you created should be adjusted to use capital characters (per these definitions), but the routes that appear in the browser address bar are correct as all lower case:
 Create a new route for "How to use We Vote": /routes/More/HowToUse.jsx (/more/howtouse)
 Create a new route for "Vision": /routes/More/Vision.jsx (/more/vision)
 Create a new route for "Organization": /routes/More/Organization.jsx (/more/organization)
 Create a new route for "Team": /routes/More/Team.jsx (/more/team)

Please take the Terms of Use text out of these four new pages (so they are empty of content). I will fill them with content from the current about page.

**Comment by Anoninnyc** [comment]
@DaleMcGrew Pls take a look at the updated PR

**Comment by DaleMcGrew** [comment]
Thank you @Anoninnyc!

**Pull request merged**